### ISSUE TITLE
Peter M. and Isabell P. would like the same rights as Evelyn P. to edit the ROW SharePoint

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#5908]
> What application are you using?

Other / Not Sure

> Describe the problem.

Peter M. and Isabell P. would like the same rights as Evelyn Po.to edit the ROW SharePoint pages.  We do not need to be "owners", unless that's required to edit the pages. Thanks!

> How soon do you need this?

Soon — This week 

> Requested By
Peter M.

Request ID: DTS21-101940

**Comment by dianamartin** [comment]
I checked the ROW SharePoint site and I had set the permissions so that `Everyone except external users` can edit/modify things. Evelyn doesn't have any special permissions that I see.

**Comment by dianamartin** [comment]
I reached out to Peter to let him know that there were no special permissions. I got on a call with him this afternoon 4/26, we included Evelyn and found out that she was able to:
- Edit a page
- Add an app
- See Site Pages 

- Peter told me that Paloma wants to create new pages for SOP processes ex. Excavation and then including all that a reviewer would need to do an Excavation permit. 
- I told Peter to let me know if they get to a point where they can't edit something and I can change their permissions
